File: Problem_plecakowy/Problem_plecakowy/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problem_plecakowy
{
    internal class Item
    {
        public int weight;
        public int value;
        public int index;
        //public double ratio;

        public Item() 
        {
            weight = 0;
            value = 0;
            index = 0;
            //ratio = 0;
        }

        public Item(int w, int v, int i) 
        { 
            weight = w;
            value = v;
            index = i;
            //ratio = (double)value / weight;
        }

        public override string ToString()
        {
            return $"V: {value} W: {weight}";
        }

       
    }
}

File: Knapsack263516/src/main/java/problempackage/Item.java
package problempackage;

public class Item {
    private int index;
    private int weight;
    private int value;

    public Item(int index, int weight, int value) {
        this.index = index;
        this.weight = weight;
        this.value = value;
    }

    public int getWeight() {
        return weight;
    }

    public int getValue() {
        return value;
    }
//    public int getIndex() {
//        return index;
//    }

    @Override
    public String toString() {
        return "No: [" + index + "] v: " + value + " w: " + weight;
    }
}

